Firestore/Example/Tests/Integration/API/FIRQueryTests.mm

```diff
@@ -974,19 +988,34 @@ - (void)testOrQueriesWithInAndNotIn {
   }];
 
   // a==2 || b in [2,3]
-  FIRFilter *filter1 = [FIRFilter orFilterWithFilters:@[
+  FIRFilter *filter = [FIRFilter orFilterWithFilters:@[
     [FIRFilter filterWhereField:@"a" isEqualTo:@2], [FIRFilter filterWhereField:@"b" in:@[ @2, @3 ]]
   ]];
-  [self checkOnlineAndOfflineQuery:[collRef queryWhereFilter:filter1]
+  [self checkOnlineAndOfflineQuery:[collRef queryWhereFilter:filter]
                      matchesResult:@[ @"doc3", @"doc4", @"doc6" ]];
+}
+
+- (void)testOrQueriesWithNotIn {
+  XCTSkipIf(![FSTIntegrationTestCase isRunningAgainstEmulator],
+            "Skip this test if running against production because order-by-equality is not "
+            "supported yet.");
```

Comment:
The message is wrong. This one is not order-by-equality.

```java
    assumeTrue(
        "Skip this test if running against production because it results in a "
            + "'missing index' error. The Firestore Emulator, however, does serve these "
            + " queries",
        isRunningAgainstEmulator());
```